redaxo/src/addons/install/pages/packages.update.php

```diff
@@ -225,7 +226,7 @@
                 <td data-title="' . $package->i18n('key') . '"><a class="rex-link-expanded" href="' . $url . '">' . rex_escape($key) . '</a></td>
                 <td data-title="' . $package->i18n('name') . '">' . rex_escape($addon['name']) . '</td>
                 <td data-title="' . $package->i18n('existing_version') . '">' . $packageVersion . '</td>
-                <td data-title="' . $package->i18n('available_versions') . '">' . implode(', ', $availableVersions) . '</td>
+                <td data-title="' . $package->i18n('available_versions') . '">' . implode(' - ', $availableVersions) . '</td>
```

Comment:
```suggestion
                <td data-title="' . $package->i18n('available_versions') . '">' . implode(' ', $availableVersions) . '</td>
```

Ich würde die nur durch Leerzeichen trennen. Aktuell sah es (bei genau zwei Versionen) nach einem Bis-Strich aus, also nach einer Versions-Range.